### ISSUE TITLE
Web-build of archipelago.js fails to import in browsers.

### ISSUE THREAD

**Issue by boiscljo** [ThePhar/archipelago.js#192]
When importing from browser, results in a empty module with no exports. 

If using the example 
```js
 import {
                Client,
                ITEMS_HANDLING_FLAGS,
                SERVER_PACKET_TYPE,
            } from "https://unpkg.com/archipelago.js/dist/archipelago.min.js";
```
The result is "The module does not contains an export named Client"

When using import * as AP from "...". Result is a module with nothing except the name. 

Nodejs version works perfectly, issue only affect browsers

**Comment by maxwelldemonic** [comment]
I'm having this exact same issue. I'm trying to build a browser based tracking solution for a game and it simply won't import properly.

**Comment by ThePhar** [comment]
This issue should be completely resolved by v2.0, which I'm releasing RC1 for today. If it isn't too much effort, I'd recommend giving the new version a try.

**Comment by ThePhar** [comment]
I just published 1.2.0 on npm that should fix this issue for legacy users. If you don't mind letting me know if that fixed it for you, I would greatly appreciate it.
